data_table.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import numpy as np
+import dash
+from dash import dash_table, html
+
+df= pd.read_csv('csv/customers-1000.csv')
+df["Subscription Date"] = pd.to_datetime(df["Subscription Date"]).dt.date
```

Comment:
It a good practice to be consisten with the string character definition, for example use always `'` apostrophe or `"` the quotation mark. 

In this case, it's being used the quotation mark, but on path it use the `'`, the code would not behave different, but it add consistency to code base.

In general, it's preferred to use `'` .